src/bootstrap/src/core/config/config.rs

```diff
@@ -114,6 +114,7 @@ pub struct Config {
     pub bootstrap_cache_path: Option<PathBuf>,
     pub extended: bool,
     pub tools: Option<HashSet<String>>,
+    pub tool_config: HashMap<String, ToolConfig>,
```

Comment:
I would rename this to just `tool`. Could you please also add a short doc comment on top of the field to explain what it does? I know most other fields don't have comments, but let's improve the current situation :)

Reply:
Done